crates/solidity/outputs/cargo/tests/src/bindings_assertions/assertions.rs

```diff
@@ -99,24 +105,22 @@ impl fmt::Display for ReferenceAssertion {
         if let Some(version_req) = &self.version_req {
             write!(f, " in versions {version_req}")?;
         }
-        write!(f, " {}", DisplayCursor(&self.cursor))
+        write!(f, " {}", DisplayCursor(&self.cursor, Some(self.file)))
     }
 }
 
-struct DisplayCursor<'a>(&'a Cursor);
+struct DisplayCursor<'a>(&'a Cursor, Option<&'a str>);
 
 impl<'a> fmt::Display for DisplayCursor<'a> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let offset = self.0.text_offset();
-        let range = self.0.text_range();
         write!(
             f,
-            "`{}` at {}:{} [{}..{}]",
+            "`{}` at {}:{}:{}",
             self.0.node().unparse(),
+            self.1.unwrap_or("<unknown_file>"),
             offset.line + 1,
```

Comment:
unrelated to this PR:
`line` and `column` should be already one-based. is this not the case here? :/

Reply:
IIUC, `TextIndex`'s `line` and `column` are [0 based](https://github.com/manastech/slang/blob/feat%2Fimport-resolver/crates/codegen/runtime/cargo/src/runtime/language/parser_support/context.rs#L27).

Reply:
I checked the [LSP specification](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#range), and looks like it is also using zero-based lines and columns. Thanks for double checking.